web-marketplace/src/components/templates/ProjectDetails/ProjectDetails.tsx

```diff
@@ -431,6 +454,13 @@ function ProjectDetails(): JSX.Element {
               : undefined
           }
           isPrefinanceProject={isPrefinanceProject}
+          onClickCreatePost={() => {
+            setIsCreatePostModalOpen(onChainProjectId || offChainProject?.id);
+          }}
+          isCreatePostButtonDisabled={
+            !projectMetadata?.['schema:location'] || !isProjectPublished
+          }
+          tooltipText={CREATE_POST_DISABLED_TOOLTIP}
```

Comment:
from now on, when adding new strings that are meant to be displayed/translated, could you use lingui? this way it's less work for devs that are working on the Terrasos project and if some component already got translated, this won't cause untranslated text.
For instance, declare the variable **in web-marketplace** as `const CREATE_POST_DISABLED_TOOLTIP = msg`some text`;` then use it like this `_(CREATE_POST_DISABLED_TOOLTIP)`

Reply:
Sure, I’ll start using Lingui. Thanks for the details!